src/components/ebay-menu-button/index.marko

```diff
@@ -55,17 +60,26 @@ $ var isOverflowVariant = input.variant === "overflow";
                 input.textAlign === 'center' && "expand-btn__cell--center"
             ]>
                 <if(input.label)>
-                    <${input.label.renderBody}/>
+                    <if(input.prefixLabel)>
+                        <span class="menu-button-prefix-label">${input.prefixLabel}</span>
+                        <${input.label.renderBody}/>
+                    </if>
+                    <else >
+                        <span id=labelId>
+                            <${input.label.renderBody}/>
+                        </span>
+                    </else>
```

Comment:
I don't think this is all needed.
You can probably wrap the if statement with a span like this:
```
<${labelId ? "span" : null} id=labelId>
   <if>
   <else-if>
</>
```
That way, if labelId is set, then it gets rendered and wraps all the possible text with the span.

Reply:
@austingardner I'm okay with what we have for now. I find it readable. If you think Andrews suggestion would be equally as readable, then go ahead and make the change. Otherwise, I'm fine with it as is.